### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by abeltest2**
Nothing to see here, just testing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # testing
 Just for testing the new rename feature
+
+More text here
```
Comment (abeltest2) [inline README.md:4]:
```suggestion
More text here.
```

**Comment by abeltest2** [comment]
Just leaving some extra comment

**Comment by abelbraaksma** [comment]
Nice change, thank you! Can you try update the source branchname?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
 # testing
 Just for testing the new rename feature
+
+More text here.
```
Comment (abelbraaksma) [inline README.md:4]: Good work!

**Review by abelbraaksma — APPROVED** [review]

**Comment by abeltest2** [comment]
Yes, let me try that

**Pull request closed without merging**